packaging/package.yml/en.md

```diff
@@ -44,44 +42,43 @@ install    : |
+**component** | `string` | Component / group of packages this package belongs to. Check available components via `eopkg lc`.
+**summary** | `string` | Brief package summary, or display name.
+**description** | `string` | More extensive description of the software, usually taken from the vendor website.
 
 ### Optional, supported keys
 
 Key Name | Type | Description
 ---- | ---- | ----
-**clang** | `bool` | Set to `no` if this package cannot be built with Clang
+**clang** | `bool` | Set to `yes` if this package benefits from being built with Clang.
 **extract** | `bool` | Set to `no` to disable automatic source extraction.
-**autodep** | `bool` | Set to `no` to disable automatic binary dependency resolution at build time
-**emul32** | `bool` | Set to `yes` to enable an `-m32` build (32-bit libs)
-**libsplit** | `bool` | Set to `no` to disable splitting of libraries into `devel` sub-packages
+**autodep** | `bool` | Set to `no` to disable automatic binary dependency resolution at build time.
+**emul32** | `bool` | Set to `yes` to enable an `-m32` build (32-bit libs).
+**libsplit** | `bool` | Set to `no` to disable splitting of libraries into `devel` sub-packages.
+**conflicts** | `string(s)` | Specify packages that cannot be installed together with this one.
 **optimize** | `list` | Specify preset keys to modify compiler and linker flags during build. You can learn more [here](/articles/packaging/package.yml/en/#optimize-values).
 **builddeps** | `list` | Specify build dependencies for the package. You can learn more [here](/articles/packaging/packaging-practices/en/#build-dependencies).
 **rundeps** | `dict(s)` | Specify further runtime dependencies for the packages. You can learn more [here](/articles/packaging/packaging-practices/en/#runtime-dependencies).
-**replaces** | `dict(s)` | Replace one package with another, used when renaming or deprecating packages for clean upgrade paths
-**patterns** | `dict(s)` | Allows fine grained control over file placement within the package or sub-packages. Useful for packages that are development only (i.e. `/usr/bin` files)
+**replaces** | `dict(s)` | Replace one package with another, used when renaming or deprecating packages for clean upgrade paths.
+**patterns** | `dict(s)` | Allows fine grained control over file placement within the package or sub-packages. Useful for packages that are development only (i.e. `/usr/bin` files).
 **environment** | `unicode` | Specify code that will be exported to all build steps of the build (i.e. exporting variables for the entire build).
-**networking** | `bool` | Set to `yes` to enable networking within solbuild
-**homepage** | `string` | Provides a link to the package's hompage in the software center.
+**networking** | `bool` | Set to `yes` to enable networking within solbuild.
+**homepage** | `string` | Provides a link to the package's hompage in the Software Center.
```

Comment:
You misspelled homepage

Reply:
Not me actually, it's a typo I didn't spot.